scripts/scout.js

```diff
@@ -8,4 +10,4 @@
  */
 
 require('../src/setup_node_env');
-require('@kbn/scout').startServersCli();
+void require('@kbn/scout').cli.run();
```

Comment:
Why are we using the void operator explicitly here?

Reply:
But void is an eval operator.  
Also, what do you mean my restricted syntax?

Reply:
> But void is an eval operator.

Yep. It evaluates the expression and ditches the returned `Promise<void>`. No promise returned, no warning in IDEA. 😊 

> Also, what do you mean by restricted syntax?

<img width="545" alt="image" src="https://github.com/user-attachments/assets/5df45dbb-82da-4fd1-bbde-f655a826678e">

Reply:
Oh I see now, you're getting around needing to use and IIFE.  
Pretty cool.